<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelEstado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlfabeto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabeto</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadoInicial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado I</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadoAceptacion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado A</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstado">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlfabeto">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstadoInicial">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstadoAceptacion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddEstado">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddAlfabeto">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okEstado">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okEstadoInicial">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddEstadoInicial">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okAlfabeto">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okEstadoAceptacion">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddEstadoAceptacion">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="Line" name="lineVertical1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>20</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstados">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>242</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estados</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlfabetos">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>271</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabeto's</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaquina">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>221</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Maquina</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstados">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>242</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadosAceptacion">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>329</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlfabetos">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>271</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstadosAceptaciones">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>329</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEstadosIniciales">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>300</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelEstadosIniciales">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>300</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estados I</string>
    </property>
   </widget>
   <widget class="Line" name="lineVertical2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>16</width>
      <height>481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="lineHorizontal2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelCadena">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadena</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCadena">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVerificarCadena">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>450</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewTransiciones">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>50</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTablaTran">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Input - Tabla de Transiciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLlenarTransiciones">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>340</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Llenar Transiciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOkeyLlenadoTransiciones">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>340</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNodo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>25</y>
      <width>641</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1355</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Dot"/>
    <addaction name="action_Iniciar"/>
    <addaction name="action_Clear"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_Help"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Dot"/>
   <addaction name="action_Iniciar"/>
   <addaction name="action_Clear"/>
   <addaction name="action_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Dot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/dot.png</normaloff>:/Resources/dot.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Dot</string>
   </property>
  </action>
  <action name="action_Iniciar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/iniciar.jpeg</normaloff>:/Resources/iniciar.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Iniciar</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/clear.jpeg</normaloff>:/Resources/clear.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/close.jpeg</normaloff>:/Resources/close.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/about.jpeg</normaloff>:/Resources/about.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
